CHANGELOG.md

```diff
@@ -5,8 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
 ### Added
+- `conjurctl server` and `conjurctl account create` allow the operator to specify the admin user's
+  password via STDIN by providing the `--password-from-stdin` switch.
+  [cyberark/conjur#2043](https://github.com/cyberark/conjur/issues/2043)]
+- `conjurctl account create` now allows the operator to specify the account name via the `--name` flag.
+  We recommend using this explicit flag when using the `--password-from-stdin` option so that commands are explicit and more readable
```

Comment:
is this really why? this was different than we talked about, though also valuable. we talked about potential issues with the order of the flags / subcommands (ie it could break if you don't use it this way, it's not just for readability)

Reply:
nit: also, this line is long

Reply:
Options always come before arguments. I thought `conjurctl account create --password-from-stdin --name <account name>` and `conjurctl account create --name <account name> --password-from-stdin` read better than `conjurctl account create --password-from-stdin <account name>`. Open to input!